source/reference/minio-server/minio-server.rst

```diff
@@ -425,6 +425,14 @@ MinIO Console:
    If you are using a standalone MinIO Console process, this variable
    corresponds with ``CONSOLE_PROMETHEUS_JOB_ID``.
 
+.. envvar:: MINIO_PROMETHEUS_AUTH_TOKEN
+
+   *Optional*
+
+   Specify the auth token the Console should use to connect to a Prometheus service.
```

Comment:
Is this just https://prometheus.io/docs/prometheus/latest/configuration/https/ ? Can we give an example of the token? maybe https://prometheus.io/docs/guides/basic-auth/

@bexsoft for guidance.

Reply:
Hi @ravindk89!, This env var is to configure a standard Bearer token only, basically the correct documentation would be `https://prometheus.io/docs/guides/basic-auth/` but in this case Prometheus uses a Basic Authentication. 

an example of a bearer token could be: `eyJhbGciOiJIUzUxMiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJwcm9tZXRoZXVzIiwic3ViIjoibWluaW8iLCJleHAiOjQ4NTAwMzg0MDJ9.GZCKR3d0FH2TCvNHSd39HaVfSuQVVV0s8glICBDmhT51V6CQ_hw8gTYlKHJmcpR8aHkqiJwCqcYJhaMmqwe00XY`

The generation of this token will depend of customer's setup and configuration